pkg/api/grpc/echo.go

```diff
@@ -0,0 +1,17 @@
+package grpc
+
+import (
+	"context"
+	"log"
+
+	"github.com/stefanprodan/podinfo/pkg/api/grpc/echo"
+)
+
+type echoServer struct {
+	 echo.UnimplementedEchoServiceServer
+}
+
+func (s *echoServer) Echo (ctx context.Context, message *echo.Message) (*echo.Message, error){
+	log.Printf("Received message body from client: %s", message.Body)
```

Comment:
We need to use the server logger instead of default one.

Reply:
Alright